app/code/Magento/User/view/adminhtml/web/app-config.js

```diff
@@ -27,7 +27,6 @@ require.config({
         'jquery/validate': 'jquery/jquery.validate',
         'jquery/hover-intent': 'jquery/jquery.hoverIntent',
         'jquery/file-uploader': 'jquery/fileUploader/jquery.fileuploader',
-        'prototype': 'prototype/prototype-amd',
```

Comment:
Could you explain, why was it removed? Tbh it's not clear why it was ever needed

Reply:
Hello @ihor-sviziev  , 
lib/web/prototype/prototype-amd.js  is empty, and it was empty back in version 2.0.
This is the only mention of it in the current codebase, so we decided to remove it.
Maybe that .js file came from M1.

Now, after your question, I'm thinking maybe we don't need this app-config.js file at all.
https://github.com/magento/magento2/commit/12a9f66ec6dce93db33bc3e342c39b14c8d4e994
Looks like there was a task to move everything from app-config.js to requirejs-config, but the file itself wasn't deleted. What do you think?

Reply:
I think we should remove it